models/vrp.rb

```diff
@@ -407,6 +407,12 @@ def self.remove_unnecessary_relations(hash)
            vehicle_group_duration_on_months vehicle_group_number]
 
       hash[:relations].delete_if{ |r| r[:lapse].nil? && types_with_duration.include?(r[:type]) }
+
+      uniq_relations = []
+      hash[:relations].group_by{ |r| r[:type] }.each{ |_type, relations_set|
+        uniq_relations += relations_set.uniq
+      }
+      hash[:relations] = uniq_relations
```

Comment:
what is the goal here, if there is a duplicate relation, then it is an invalid vrp (i.e., DiscordantProblemError)? Like the way, it is invlaid when there is a duplicate service.

Reply:
From my point of view if it is twice the exact same relation I do not think we need to reject it because if we can satisfy it once we can satisfy it twice. Result will be consistent with what data provided by user. 

But at the same time it could help user notice he provided 'weird' data.

Reply:
I think your last phrase is more important that is why I think we should raise otherwise the user can't know.